README.md

```diff
@@ -30,3 +30,28 @@ function mystery(n) {
 Add your answer to this markdown file. [This
 page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
 might help with the notation for mathematical expressions.
+
+## Recurrence Relation
+
+This function calls mystery() three times. It calls the exact same value of mystery(n/3) regardless of the time, which means each call gets called a further three times until n<=1. 
+
+For each call of mystery, the for loop with k is called $n^2$ times, the for loop with j is called $n$ times, and the for loop for i is called $n^2$ times. $n^2 * n * n^2 = n^5$
+
+So the recurrence relation can be written as follows: $3T(n/3) + n^5$
+
+## Time Complexity
+
+To find the time complexity, we find a form after a few iterations
+$3(3T(n/9) + (n/3)^5) + n^5 = 9T(n/9) + 3(n/3)^5 + n^5 $
+
+$9(3T(n/27) + (n/9)^5) + n^5 + 3(n/3)^5 = 27T(n/27) + 9(n/9)^5 + 3(n/3)^5 + n^5$
+
+In each case, the greatest power is $n^5$ so considering the rest of the function is the worst case, we get our $O$ worst case:
```

Comment:
But how many terms do you have? This increases with the number of recursive calls -- would this not affect the complexity?